src/libstd/collections/hash/map.rs

```diff
@@ -1282,7 +1291,7 @@ pub struct IterMut<'a, K: 'a, V: 'a> {
 /// HashMap move iterator.
 #[stable(feature = "rust1", since = "1.0.0")]
 pub struct IntoIter<K, V> {
-    inner: table::IntoIter<K, V>,
+    pub(super) inner: table::IntoIter<K, V>,
```

Comment:
> Hm for now instead of pub(super) could this just be a public function in this module to get access to the table? We could then avoid exporting that as part of the hash_map module.

@alexcrichton Would this be accomplished by changing [this line](https://github.com/rust-lang/rust/blob/fa0005f2d5a9081597867b383c9b2d0b21569b88/src/libstd/collections/mod.rs#L444-L449) to explicitly export every thing _but_ your new proposed function? Otherwise, wouldn't that function also get exported?